x/profile/types/key_validator.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import "encoding/binary"
+
+var _ binary.ByteOrder
+
+const (
+	// ValidatorKeyPrefix is the prefix to retrieve all Validator
+	ValidatorKeyPrefix = "Validator/value/"
+)
+
+// ValidatorKey returns the store key to retrieve a Validator from the index fields
+func ValidatorKey(address string) []byte {
+	var key []byte
+
+	addressBytes := []byte(address)
+	key = append(key, addressBytes...)
+	key = append(key, []byte("/")...)
+
+	return key
```

Comment:
This is another example on where the template produce more complex code than needed because it need to be generate (the template must be able to create the key for an index with multiple different types)

Here we could have simply: `return []byte(address + "/")`

This is not blocking for the PR because I will note that all `<Type>Key(...)` methods will have to be reviewed